blog/app/models/comment.rb

```diff
@@ -0,0 +1,3 @@
+class Comment < ActiveRecord::Base
+  belongs_to :article
```

Comment:
Updated it to validate that there is a commenter and a body. I can validate for the article, but since I can't access a page without an article, is that needed? Also, if the comment section would be reused there might not be an article associated with it.